pkg/kubectl/categories/categories.go

```diff
@@ -129,12 +140,16 @@ func (e discoveryFilteredExpander) Expand(category string) ([]schema.GroupResour
 	return available, ok
 }
 
+// UnionCategoryExpander represents for a list of CategoryExpander,
+// expends to a union of gourpResources lists, which merges all expentions of CategoryExpander in UnionCategoryExpander.
 type UnionCategoryExpander []CategoryExpander
 
+//Expand returns a single list of all GroupResource objects. If not found, return empty slice and false
 func (u UnionCategoryExpander) Expand(category string) ([]schema.GroupResource, bool) {
 	ret := []schema.GroupResource{}
 	ok := false
 
+	// Traverse the UnionCategoryExpander, expend for each CategoryExpander,
```

Comment:
may be say: expand the category for each CategoryExpander in the list and merge/combine the results.

Reply:
Done